mltsp/science_feature_tools.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import cfg
+import science_features as sf
+from dask.async import get_sync as dget
```

Comment:
rather use something like ``import dask.async as da`` and then use ``da.get_sync``, otherwise it's hard to figure out below where ``dget`` comes from.